pygmo/_py_algorithms.py

```diff
@@ -0,0 +1,131 @@
+import random
+
+try:
+    from scipy.optimize import minimize
+
+    class scipy:
+        """
```

Comment:
For the docstring its good to use the same style of all others (see the population class for example): https://github.com/esa/pygmo2/blob/master/pygmo/docstrings.cpp#L304

including the various Args and Raises when needed (init) and possibly some code blocks explaining the use

Reply:
Addressed in commit a912aed